.env.template

```diff
@@ -0,0 +1 @@
+MAPS_API_KEY=XXXXXXXXXXXXXXXXXXX
```

Comment:
Why not putting the Github token?

Reply:
the google maps token is used inside the native app and the github token is used in the dart code. Since the .env file is only used on the dev machine it would need additional code in the app itself to manage the github token in that file.

Reply:
Ohhh @AlexandreMarkus I think I found something quicker than what I sent you before! https://artem-diashkin.medium.com/flutter-using-environment-variables-24a976ae1335. The only drawback is that instead of calling `String.FromEnvironment('GITHUB_API_TOKEN')`, we will have to use their custom object from the library `DotEnv().env['GITHUB_API_TOKEN']`.